src/axom/core/IteratorBase.hpp

```diff
@@ -9,15 +9,11 @@
  * \brief Contains iterator base classes
  */
 
-#ifndef SLAM_ITERBASE_H_
-#define SLAM_ITERBASE_H_
-
-#include "axom/slam/Set.hpp"
+#ifndef ITERBASE_H_
+#define ITERBASE_H_
```

Comment:
Please prefix the header include guards with `AXOM_`, i.e., `AXOM_ITERBASE_HPP_`